src/State/taskActions.ts

```diff
@@ -0,0 +1,46 @@
+export const ADD_TASK = "ADD_TASK";
+export const REMOVE_TASK = "REMOVE_TASK";
+export const EDIT_TASK = "EDIT_TASK";
+export const TOGGLE_TASK_COMPLETION = "TOGGLE_TASK_COMPLETION";
+
+export const ADD_TASK_STARTED = "ADD_TASK_STARTED";
+export const ADD_TASK_SUCCESS = "ADD_TASK_SUCCESS";
+export const ADD_TASK_FAILED = "ADD_TASK_FAILED";
+
+export const REMOVE_TASK_STARTED = "REMOVE_TASK_STARTED";
```

Comment:
same here please remove unused code or explain why its here

Reply:
@faizanmunir017 please make this an enum or interface if it is still needed.